app/views/pages/index.html.haml

```diff
@@ -1 +1,34 @@
-= render 'layouts/nav'
+= render :partial => 'layouts/nav'
```

Comment:
`render "layouts/nav"` will do, `:partial` is not needed. Also, strive for consistency sa quotes.